dashboard/app/helpers/levels_helper.rb

```diff
@@ -672,7 +673,8 @@ def render_multi_or_match_content(text)
     return match_answer_as_embedded_blockly(path) if File.extname(path).ends_with? '_blocks'
     return match_answer_as_iframe(path, width) if File.extname(path) == '.level'
 
-    text
+    @markdown_renderer ||= Redcarpet::Markdown.new(Redcarpet::Render::Inline)
+    @markdown_renderer.render(text).html_safe
```

Comment:
This looks like it will create a new markdown renderer for each instance of each class which includes this module and calls this method, including once for each multi or match script level object. Perhaps it would be better to store it on `@@markdown_renderer`, so that it only gets instantiated once for each class which includes this module and calls this method?

Reply:
oh, good catch!